src/views/ProfileCreation/UserName.tsx

```diff
@@ -81,28 +83,56 @@ const UserName: React.FC = () => {
   )
   const isUserCreated = existingUserState === ExistingUserState.CREATED
 
-  const checkUsernameValidity = debounce(async (value: string) => {
-    try {
-      setIsLoading(true)
-      const res = await fetch(`${API_PROFILE}/api/users/valid/${value}`)
+  const [usernameAvailability, setUsernameAvailability] = useState<string>(undefined)
```

Comment:
`usernameAvailability` is basically just the string typed by user? Should probably rename to something less ambiguous, like "username" or "usernameToCheck". 🤔

Reply:
Done